Makefile

```diff
@@ -42,6 +42,10 @@ run-flask: ## Run flask
 run-flask-with-docker: ## Run flask
 	./scripts/run_locally_with_docker.sh api-local
 
+.PHONY: run-local-db-with-docker
+run-local-db-with-docker: ##
+	./scripts/run_local_db_with_docker.sh api-local
```

Comment:
Do we need a whole new script or can we just add a new target to the `run_locally_with_docker` script, eg `api-local-db`?

Reply:
@samuelhwilliams i have made smallish changes to the run_locally_with_docker script. Please see if these are acceptable, in that case, we can run with just that one script

Reply:
Please have a review of the original task on https://trello.com/c/SRDI5dT8/576-get-api-unit-tests-to-run-in-the-api-docker-image-not-the-generic-unit-test-image-on-concourse